chatwoot.yaml

```
@@ -11,51 +11,133 @@ common:
     source: https://github.com/chatwoot/chatwoot
     publisher: "monk.io"
     icon: https://storage.googleapis.com/template_logos/chatwoot-image.png
-
+    email: maciej@monk.io
```

we can update the email. 

```suggestion
    email: hello@chatwoot.com
```